### ISSUE TITLE
CAPTCHA error

### ISSUE THREAD

**Issue by onezetty** [Heroic-Games-Launcher/HeroicGamesLauncher#2654]
### Describe the bug

Every time I try to sing up at Epic the logging fails because the CAPTCHA. I can confirm my password, it is correct. 

### Add logs

```text
(20:59:26) INFO:    [Connection]:       Connectivity: check-online
(20:59:26) INFO:    [Connection]:       Pinging external endpoints
(20:59:27) INFO:    [Legendary]:        Running command: /opt/Heroic/resources/app.asar.unpacked/build/bin/linux/legendary --version
(20:59:27) INFO:    [Legendary]:        Legendary location: /opt/Heroic/resources/app.asar.unpacked/build/bin/linux/legendary
(20:59:27) INFO:    [Gog]:              GOGDL location: /opt/Heroic/resources/app.asar.unpacked/build/bin/linux/gogdl
(20:59:27) INFO:    [Gog]:              GOGDL location: /opt/Heroic/resources/app.asar.unpacked/build/bin/linux/gogdl
(20:59:27) WARNING: [Backend]:          Protocol already registered.
(20:59:27) INFO:    [Gog]:              Running command: /opt/Heroic/resources/app.asar.unpacked/build/bin/linux/gogdl --auth-config-path /home/onezetty/.config/heroic/gog_store/auth.json --version
(20:59:27) INFO:    [Connection]:       Connectivity: online
(20:59:27) INFO:    [Backend]:          User Not Found, removing it from Store
(20:59:27) INFO:    [Gog]:              Getting data about the user
(20:59:27) INFO:    [Gog]:              Running command: /opt/Heroic/resources/app.asar.unpacked/build/bin/linux/gogdl --auth-config-path /home/onezetty/.config/heroic/gog_store/auth.json auth
(20:59:27) INFO:    [Backend]:          AreWeAntiCheatYet data downloaded
(20:59:27) INFO:    [Backend]:          Loading Screen Ready
(20:59:27) INFO:    [Frontend]:         Refreshing Library
(20:59:27) INFO:    [Legendary]:        Refreshing library...
(20:59:27) ERROR:   [Legendary]:        Platform undefined was not found in assets.json?
(20:59:27) INFO:    [Legendary]:        Found 0 games to update
(20:59:27) INFO:    [Gog]:              Running command: /opt/Heroic/resources/app.asar.unpacked/build/bin/linux/gogdl --auth-config-path /home/onezetty/.config/heroic/gog_store/auth.json auth
(20:59:27) INFO:    [Backend]:          Frontend Ready
(20:59:27) INFO:    [Backend]:          

Heroic Version: 2.7.0 Eustass Kid
Legendary Version:  0.20.32 Dark Energy (hotfix #6)
GOGdl Version: 0.7.1

Electron Version: 23.1.4
Chrome Version: 110.0.5481.192
NodeJS Version: 18.12.1

OS: Ubuntu KERNEL: 5.8.18-050818-generic ARCH: x64
CPU: Intel Core™ i5-8300H @2.3 GOVERNOR: powersave
RAM: Total: 23.33 GiB Available: 17.87 GiB
GRAPHICS: GPU0: UHD Graphics 630  VRAM: 256MB GPU1: NVIDIA GeForce GTX 1050 Ti with Max-Q Design VRAM: 4096MB DRIVER: 525.105.17
PROTOCOL: x11

(20:59:27) INFO:    [Backend]:          Checking for new Heroic Updates
(20:59:27) INFO:    [Backend]:          Checking for current version changelog
(20:59:28) INFO:    [Gog]:              Getting GOG library
(20:59:28) INFO:    [Gog]:              Saved user data to config
(20:59:28) INFO:    [Gog]:              Number of library pages: 1
(20:59:28) INFO:    [Gog]:              Running command: /opt/Heroic/resources/app.asar.unpacked/build/bin/linux/gogdl --auth-config-path /home/onezetty/.config/heroic/gog_store/auth.json auth
(20:59:29) INFO:    [Gog]:              Found 0 game(s) to update
(20:59:29) INFO:    [Frontend]:         No cache found, getting data from legendary...
(20:59:29) INFO:    [Legendary]:        Refreshing library...
(20:59:29) INFO:    [Winetricks]:       Downloading Winetricks
(20:59:30) INFO:    [Gog]:              Saved games data
(20:59:32) INFO:    [Backend]:          Starting the Download Queue
```

### Steps to reproduce

2. Click on '...'
3. Scroll down to '...'

### Expected behavior

No login 

### Screenshots

![2023-04-26_20-58](https://user-images.githubusercontent.com/4399989/234741688-245a1731-58ca-44d9-9d3a-dd67581556c0.png)

### Heroic Version

Latest Stable

### System Information

OS Ubuntu 20.04

**Comment by bearcatjamboree** [comment]
I can confirm that this happens ever time for me as well.

**Comment by devexa** [comment]
Same issue here, for the first time I was logged out from Epic in Heroic and I can't log back in because of an error I get after choosing the right images in Captcha.

**Comment by flavioislima** [comment]
I can confirm the issue. For now, use the alternative method.

**Comment by Slothoncrack** [comment]
Can confirm from my steam deck.

What's the alternative method?

**Comment by zeitgeists** [comment]
When I try to **PLACE ORDER** and solve CAPTCHA correctly , I get this error: `Failed to challenge captcha, please try again later.`
 If the CAPTCHA is not solved then I get the message "try again", as it should be.
 
 Here some logs:
 ```
 (00:04:37) INFO:    [Connection]:       Connectivity: check-online
(00:04:37) INFO:    [Connection]:       Pinging external endpoints
(00:04:37) INFO:    [Legendary]:        Running command: /opt/Heroic/resources/app.asar.unpacked/build/bin/linux/legendary --version
(00:04:37) INFO:    [Legendary]:        Legendary location: /opt/Heroic/resources/app.asar.unpacked/build/bin/linux/legendary
(00:04:37) INFO:    [Gog]:              GOGDL location: /opt/Heroic/resources/app.asar.unpacked/build/bin/linux/gogdl
(00:04:37) INFO:    [Gog]:              GOGDL location: /opt/Heroic/resources/app.asar.unpacked/build/bin/linux/gogdl
libva error: vaGetDriverNameByIndex() failed with unknown libva error, driver_name = (null)
APPIMAGE env is not defined, current application is not an AppImage
(00:04:38) WARNING: [Backend]:          Protocol already registered.
[379832:0428/000438.265179:ERROR:browser_main_loop.cc(273)] Gtk: gtk_widget_add_accelerator: assertion 'GTK_IS_ACCEL_GROUP (accel_group)' failed
[379832:0428/000438.265279:ERROR:browser_main_loop.cc(273)] Gtk: gtk_widget_add_accelerator: assertion 'GTK_IS_ACCEL_GROUP (accel_group)' failed
[379832:0428/000438.265361:ERROR:browser_main_loop.cc(273)] Gtk: gtk_widget_add_accelerator: assertion 'GTK_IS_ACCEL_GROUP (accel_group)' failed
(00:04:38) INFO:    [Connection]:       Connectivity: online
(00:04:38) INFO:    [Gog]:              Running command: /opt/Heroic/resources/app.asar.unpacked/build/bin/linux/gogdl --auth-config-path /home/bluten/.config/heroic/gog_store/auth.json --version
(00:04:38) INFO:    [Backend]:          AreWeAntiCheatYet data downloaded
/bin/sh: line 1: hostname: command not found
(00:04:39) INFO:    [Backend]:          Loading Screen Ready
(00:04:39) INFO:    [Backend]:

Heroic Version: 2.7.0 Eustass Kid
Legendary Version:  0.20.32 Dark Energy (hotfix #6)
GOGdl Version: 0.7.1

Electron Version: 23.1.4
Chrome Version: 110.0.5481.192
NodeJS Version: 18.12.1

OS: Arch KERNEL: 6.2.12-arch1-1 ARCH: x64
CPU: AMD FX(tm)-8300 Eight-Core Processor @3.3 GOVERNOR: performance
RAM: Total: 7.72 GiB Available: 3.32 GiB
GRAPHICS: GPU0: NVIDIA GeForce GTX 1050 Ti VRAM: 4096MB DRIVER: 530.41.03
PROTOCOL: x11

(00:04:39) INFO:    [Frontend]:         Refreshing Library
(00:04:39) INFO:    [Legendary]:        Refreshing library...
(00:04:39) INFO:    [Legendary]:        Refreshing Epic Games...
(00:04:39) INFO:    [Legendary]:        Found 0 games to update
(00:04:39) INFO:    [Legendary]:        Game list updated, got 323 games & DLCs
(00:04:39) INFO:    [Gog]:              Found 0 game(s) to update
(00:04:39) INFO:    [Backend]:          Frontend Ready
(00:04:39) INFO:    [Backend]:          Checking for current version changelog
(00:04:39) INFO:    [Legendary]:        Running command: /opt/Heroic/resources/app.asar.unpacked/build/bin/linux/legendary list --third-party
(00:04:40) INFO:    [Winetricks]:       Downloading Winetricks
(00:04:44) INFO:    [Backend]:          Starting the Download Queue
 ```
 
 Screenshot:
![image](https://user-images.githubusercontent.com/77320310/234995911-831be948-a99e-4c89-b0f4-772c78365183.png)

**Comment by flavioislima** [comment]
Yes, the problem here is that we are using an old user agent.
We force a user agent so we can avoid the problem of 'platform not supported' on Linux when adding games, but it got too old and the problem is not supported by re-Captcha anymore.

Fixing this one at #2658
